### DISCUSSION TITLE
可以帮忙打包一个软件么

### DISCUSSION THREAD

**Discussion by weqoocu** [tw93/Pake#703] [Q&A]
https://marker.dotalk.cn/#/daily

**Comment by weqoocu** [comment]
有人么

**Comment by tw93** [comment]
https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89
用这个方案，不会技术也可以打包成功～

**Comment by weqoocu** [comment]
fork后，github也没有生成
<img width="882" alt="image" src="https://github.com/user-attachments/assets/e6421ef3-31ed-4983-8c7c-4ef67a75abd8">
